src/server.c

```diff
@@ -58,7 +58,20 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
     ///////////////////
     // IMPLEMENT ME! //
     ///////////////////
+    int response_length = 0;
+    time_t rawtime;
+    struct tm *timeinfo;
 
+    time(&rawtime);
+
+    timeinfo = localtime(&rawtime);
+
+    // Build HTTP response and store it in response
+    response_length = sprintf(response, "%s\nConnection: close\nContent-Length: %d\nContent-Type: %s\nDate: %s\n", header, content_length, content_type, asctime(timeinfo));
+
+    memcpy(response+response_length, body, content_length);
```

Comment:
very nice! this will let you render jpgs and gifs. nice catch on setting `response+response_length` as well